R/get-pit.R

```diff
@@ -0,0 +1,45 @@
+#' @title Probability integral transformation (data.frame version)
```

Comment:
There is currently nothing in the file naming to indicate when somerthing is a normal function vs a s3 default method. We might want to do this to make it ewasieer for people to find things which they needed to write new methods for.